papis/importer.py

```diff
@@ -66,7 +66,7 @@ def match_data(cls, data: Dict[str, Any]) -> None:
         raise NotImplementedError(
             "Matching data not implemented for this importer")
 
-    def fetch(self) -> str:
+    def fetch(self) -> Any:
```

Comment:
Not quite sure about this change. I changed it to `Any` because the docs say it could return a dict, but the method in `ArxividFromPdfImporter` doesn't return anything.